TerminalDocs/customize-settings/profile-appearance.md

```diff
@@ -28,14 +28,30 @@ The settings listed below affect the visual settings of each profile separately.
 
 This is the name of the color scheme used in the profile. Color schemes are defined in the `schemes` object. More detailed information can be found on the [Color schemes page](./color-schemes.md).
 
+This property can alternatively accept a pair of color scheme names, as follows:
+
+```json
+"colorScheme":
+{
+    "light": "One Half Light",
+    "dark": "One Half Dark",
+},
+```
+
+When specified in this manner, the Terminal will automatically switch between the two given color schemes, depending on the theme of the application. The Terminal will obey the [`theme.applicationTheme`](./themes#application-theme) property of the Terminal's selected theme. If that `applicationTheme` is set to `system`, then this will instead use the color scheme matching the theme as set by the OS.
```

Comment:
```suggestion
When specified in this manner, the Terminal will automatically switch between the two given color schemes depending on the theme of the application. The Terminal will follow the [`theme.applicationTheme`](./themes#application-theme) property of the Terminal's selected theme. If that `applicationTheme` is set to `system`, then this will instead use the color scheme matching the OS theme.
```